src/memory/allocator.hpp

```diff
@@ -181,6 +181,37 @@ namespace impl {
             }
         };
 
+        // bare bones implementation of standard compliant allocator for managed memory
+        template <size_type Alignment=1024>
```

Comment:
There is a function allignment on this class which gives back a default 256.
And not the value of this template parameter